policyengine_us/parameters/gov/states/oh/tax/income/credits/exemption/agi_less_exemption_cap.yaml

```
@@ -0,0 +1,10 @@
+description: OH Exemption Credit AGI Limit
```

Please write other parameter descriptions similarly ("Ohio [verb phrase] [this value]")
```suggestion
description: Ohio limits its exemption credit to filers with modified adjusted gross income less exemptions less than this value.
```

---

Resolved